### PR TITLE
Calculator buttons

### PR DESCRIPTION
**Pull request by BQuinteros**
Maked the desing of keypad for the calculator.

### CODE REVIEW

**Review by BQuinteros — COMMENTED** [review]
picture

File: app/src/main/java/com/example/calculadoramvp/mvp/presenter/CalculatorPresenter.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    private CalculatorModel model;
+    private CalculatorView view;
+
+    public CalculatorPresenter(CalculatorModel model, CalculatorView view){
+        this.model = model;
+        this.view = view;
+    }
+
+    public void onButtonPressed(){
+        //SI ES UN =
```
Comment (SartiMau) [inline app/src/main/java/com/example/calculadoramvp/mvp/presenter/CalculatorPresenter.java]: put all the comments in english pls

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+        app:layout_constraintEnd_toStartOf="@+id/button_multiply"
+        app:layout_constraintStart_toEndOf="@id/button_five"
+        app:layout_constraintTop_toBottomOf="@+id/button_nine"
+        app:layout_constraintBottom_toTopOf="@id/button_three"/>
+
+    <Button
+        android:id="@+id/button_seven"
+        android:layout_width="@dimen/button_layout_width"
+        android:layout_height="@dimen/button_layout_height"
+        android:layout_marginTop="150dp"
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: use dimens pls

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+        app:layout_constraintBottom_toTopOf="@id/button_four"
+        app:layout_constraintEnd_toStartOf="@+id/button_eight"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <Button
+        android:id="@+id/button_eight"
+        android:layout_width="@dimen/button_layout_width"
+        android:layout_height="@dimen/button_layout_height"
+        android:layout_marginTop="150dp"
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: same

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+        app:layout_constraintBottom_toTopOf="@id/button_five"
+        app:layout_constraintEnd_toStartOf="@+id/button_nine"
+        app:layout_constraintStart_toEndOf="@id/button_seven"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <Button
+        android:id="@+id/button_nine"
+        android:layout_width="@dimen/button_layout_width"
+        android:layout_height="@dimen/button_layout_height"
+        android:layout_marginTop="150dp"
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: same

File: app/src/main/res/layout/activity_main.xml [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+        app:layout_constraintStart_toEndOf="@id/button_six"
+        app:layout_constraintBottom_toTopOf="@id/button_subtract"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/button_share" />
+
+    <Button
+        android:id="@+id/button_share"
+        android:layout_width="@dimen/button_layout_width"
+        android:layout_height="@dimen/button_layout_height"
+        android:layout_marginTop="150dp"
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml]: same

**Review by SartiMau — CHANGES_REQUESTED** [review]

**Review by SartiMau — APPROVED** [review]

**Review by priettt — APPROVED** [review]

**Pull request merged**